ui/components/SidebarComponent.tsx

```diff
@@ -73,40 +104,46 @@ export default function SidebarComponent(props: {}) {
                   }
                 />
               </div>
-            </div>
-            <Logout />
-          </>
+              <Logout />
+            </>
+          ) : (
+            <></>
+          )
         }
         bottomLabel={
-          <div style={centerFlexStyle}>
-            <Label as='label' style={{ textAlign: 'center', fontSize: 15 }}>
-              {' '}
-              <b>Version: </b>
-              {isLoading ? 'Loading...' : version?.version}
-            </Label>
-          </div>
+          sidebarState == 'open' ? (
+            <div style={centerFlexStyle}>
+              <Label as='label' style={{ textAlign: 'center', fontSize: 15 }}>
+                {' '}
+                <b>Version: </b>
+                {isLoading ? 'Loading...' : version?.version}
+              </Label>
+            </div>
+          ) : (
+            <></>
+          )
```

Comment:
```suggestion
          sidebarState == 'open' && (
            <div style={centerFlexStyle}>
              <Label as='label' style={{ textAlign: 'center', fontSize: 15 }}>
                {' '}
                <b>Version: </b>
                {isLoading ? 'Loading...' : version?.version}
              </Label>
            </div>
          )
```

Reply:
Sidebar component's attributes don't accept `boolean | JSXElement` which is the resulting type of condition && element